# set_speed_1.msg
int32[] finger_ids   
int32[] speeds      # （0-1000）

